### PR TITLE
Bank Accounts Component - ViewModel

### PR DESCRIPTION
**Pull request by nalancer08**
### Type of change

- [X] Chore
- [ ] Feature
- [ ] Bug fix

### CODE REVIEW

File: CybridSDK/Utilities/Networking/CybridJSONDecoder.swift [hunk 1]
```diff
@@ -11,6 +11,7 @@ import Foundation
 
 final class CybridJSONDecoder: JSONDecoder {
 
+    // swiftlint:disable:next cyclomatic_complexity
```
Comment (brent-cybrid) [inline CybridSDK/Utilities/Networking/CybridJSONDecoder.swift:14]: Why do we have to do the decoding manually for all models? I know we had to do it for models where large numbers were coming back, but the deserialize that the OpenAPI generated code generates should work for the other models.

File: CybridSDKTests/Mocks/JSON/createExternalBankAccountBankModelResponse.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "guid": "stringstringstringstringstringst",
+  "name": "string",
+  "asset": "string",
+  "account_kind": "plaid",
+  "environment": "sandbox",
+  "bank_guid": "stringstringstringstringstringst",
+  "customer_guid": "stringstringstringstringstringst",
+  "created_at": "2022-12-12T17:34:58.684Z",
+  "plaid_access_token": "string"
+}
```
Comment (brent-cybrid) [inline CybridSDKTests/Mocks/JSON/createExternalBankAccountBankModelResponse.json:11]: This should have a state associated with it.

File: CybridSDK/Utilities/Networking/CybridJSONDecoder.swift (hunk 1 above)
Comment (nalancer08) [reply CybridSDK/Utilities/Networking/CybridJSONDecoder.swift:14]: Yes this is a horrible problem, but the Date and custom enums are bugged so if we don't decode manually it didn't works!

File: CybridSDKTests/Mocks/JSON/fetchWorkflowResponse.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "guid": "stringstringstringstringstringst",
+  "customer_guid": "stringstringstringstringstringst",
+  "type": "plaid",
+  "created_at": "2022-12-09T16:37:09.434Z",
+  "plaid_link_token": "string"
+}
```
[the remark this answers is not in the document]
Comment (nalancer08) [reply CybridSDKTests/Mocks/JSON/fetchWorkflowResponse.json:9]: Done!

File: CybridSDKTests/Mocks/JSON/createWorkflowResponse.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "guid": "stringstringstringstringstringst",
+  "customer_guid": "stringstringstringstringstringst",
+  "type": "plaid",
+  "created_at": "2022-12-09T16:34:13.043Z"
+}
```
[the remark this answers is not in the document]
Comment (nalancer08) [reply CybridSDKTests/Mocks/JSON/createWorkflowResponse.json:6]: Done!

File: CybridSDKTests/Mocks/JSON/createExternalBankAccountBankModelResponse.json (hunk 2 above)
Comment (nalancer08) [reply CybridSDKTests/Mocks/JSON/createExternalBankAccountBankModelResponse.json:11]: Done!

**Review by brent-cybrid — APPROVED** [review]

**Pull request merged**